mod/caching/set/type_plus_right/metric/wikirate_company.rb

```diff
@@ -5,12 +5,19 @@
 end
 
 expired_cached_count_cards :set=>Type::MetricValue, :on=>[:create,:delete] do |changed_card|
-  [
-    changed_card.metric_card.fetch(:trait=>:wikirate_company),
-    # actually this is not getting cached count for `metric+company` card
-    # it is to refresh the related metric value to the company
-    # refer to ltype_rtype/metric/wikirate_company.rb
-    Card.fetch("#{changed_card.metric_name}+#{changed_card.company_name}")
+  result = [
+    # cache number of the metric values related to this metric value set
+    # for some query
+    changed_card.metric_card.fetch(:trait=>:wikirate_company)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.